src/Qir/Execution/QirCommandLineTool/QirCommandLineTool.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20213.1" />
+    <ProjectReference Include="..\..\..\Simulation\Simulators\Microsoft.Quantum.Simulators.csproj" />
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.16.210526824-alpha" />
```

Comment:
Should this be a project reference, since it's intra-repo?

Reply:
See this comment: https://github.com/microsoft/qsharp-runtime/pull/681#discussion_r631972088

Reply:
Ah, sorry for bifurcating the discussion, then!